src/replication.c

```diff
@@ -2503,7 +2503,8 @@ int slaveTryPartialResynchronization(connection *conn, int read_reply) {
      * return PSYNC_TRY_LATER if we believe this is a transient error. */
 
     if (!strncmp(reply,"-NOMASTERLINK",13) ||
-        !strncmp(reply,"-LOADING",8))
+        !strncmp(reply,"-LOADING",8) ||
+        !strncmp(reply,"-Reading from master",20))
```

Comment:
The entire synchronization process has this problem, where we aren't really checking for read errors across all the various replication states. Like for handling the REPL_STATE_RECEIVE_CAPA_REPLY state, seems like we should fix all of them?

Reply:
Fixed